console/src/range/migrations/v0.ts

```diff
@@ -41,15 +45,13 @@ export const sliceStateZ = z.object({
   version: z.literal("0.0.0"),
   activeRange: z.string().nullable(),
   ranges: z.record(rangeZ),
-  buffer: staticRangeZ.partial().nullable(),
```

Comment:
@emilbon99 Should I create a v1.ts file with the `buffer` field removed from the `SliceState`?

Reply:
No need. I think it's fine if we leave that as stranded state.